src/index.html

```diff
@@ -5,8 +5,315 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="page__header header">
+      <a href="#homePage" class="header__logo">
+        <img src="./images/logo.png" alt="moyo-logo">
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphone-telephone" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops-computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+            Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">ФОТО</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div class="product-card catalog__product-card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="product-card__imac-image"
+          >
+
+          <h3 class="product-card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+          <p class="product-card__code">Код товара: 195434</p>
+
+          <div class="product-card__rating">
+            <div class="product-card__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="product-card__reviews">Отзывов: 5</p>
+          </div>
+
+          <div class="product-card__price">
+            <p class="product-card__price-word">Цена:</p>
+            <p class="product-card__price-digit">69 999 грн</p>
+          </div>
+
+          <a
+            href="#"
+            class="product-card__buy-link buy-link"
+            data-qa="card-hover"
+          >
+            КУПИТЬ
+          </a>
```

Comment:
Not sure that it was necessary to create separate block for this link. Using it as `product-card` element would be enough.